arm-sql/2014-04-01/examples/DatabaseCreateCopy.json

```diff
@@ -0,0 +1,76 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "sqlcrudtest-4799",
+    "serverName": "sqlcrudtest-6440",
+    "databaseName": "testdb",
+    "api-version": "2014-04-01",
+    "parameters": {
+    "location": "Japan East",
+      "properties": {
+          "createMode": "OnlineSecondary",
+          "sourceDatabaseId": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-4799/providers/Microsoft.Sql/servers/sqlcrudtest-3782/databases/testdb"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-4799/providers/Microsoft.Sql/servers/sqlcrudtest-6440/databases/testdb",
+        "name": "testdb",
+        "type": "Microsoft.Sql/servers/databases",
+        "location": "Japan East",
+        "kind": "v12.0,user",
+        "properties": {
+          "edition": "Standard",
+          "status": "Online",
+          "serviceLevelObjective": "S0",
+          "collation": "SQL_Latin1_General_CP1_CI_AS",
+          "maxSizeBytes": "268435456000",
+          "currentServiceObjectiveId": "f1173c43-91bd-4aaa-973c-54e79e15235b",
+          "requestedServiceObjectiveId": "f1173c43-91bd-4aaa-973c-54e79e15235b",
+          "requestedServiceObjectiveName": "S0",
+          "sampleName": null,
+          "defaultSecondaryLocation": "Japan West",
+          "earliestRestoreDate": "2017-02-10T01:52:52.923Z",
+          "elasticPoolName": null,
+          "containmentState": 2,
+          "readScale": "Disabled",
+          "failoverGroupId": null
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-4799/providers/Microsoft.Sql/servers/sqlcrudtest-6440/databases/testdb",
+        "name": "testdb",
+        "type": "Microsoft.Sql/servers/databases",
+        "location": "Japan East",
+        "kind": "v12.0,user",
+        "properties": {
+          "edition": "Standard",
+          "status": "Online",
+          "serviceLevelObjective": "S0",
+          "collation": "SQL_Latin1_General_CP1_CI_AS",
+          "maxSizeBytes": "268435456000",
+          "currentServiceObjectiveId": "f1173c43-91bd-4aaa-973c-54e79e15235b",
+          "requestedServiceObjectiveId": "f1173c43-91bd-4aaa-973c-54e79e15235b",
+          "requestedServiceObjectiveName": "S0",
+          "sampleName": null,
+          "defaultSecondaryLocation": "Japan West",
+          "earliestRestoreDate": "2017-02-10T01:52:52.923Z",
+          "elasticPoolName": null,
+          "containmentState": 2,
+          "readScale": "Disabled",
+          "failoverGroupId": null
+        }
+      }
+    },
+    "202": {
```

Comment:
I don't expect it to be removed from the Service API before we go GA with what we have here.  I can remove it from examples.